### PR TITLE
Approx cal for Tracklet Calculator Displaced

### PR DESCRIPTION
**Pull request by gabhijith**
#### PR description:

Approx Cal, after running checkes from https://twiki.cern.ch/twiki/bin/viewauth/CMS/L1TrackCodeRules

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Before submitting your pull requests, make sure you followed this checklist:
- verify that the PR is really intended for the chosen branch
- verify that changes follow [CMS Naming, Coding, And Style Rules](http://cms-sw.github.io/cms_coding_rules.html)
- verify that the PR passes the basic test procedure suggested in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html)

### CODE REVIEW

**Comment by gershtein** [comment]
approved

**Comment by tomalin** [comment]
@gabhijith If you've not already done so, please ensure that you've done the clang checks and eliminated the other common coding issues listed in https://twiki.cern.ch/twiki/bin/view/CMS/L1TrackCodeRules

File: L1Trigger/TrackFindingTracklet/src/TrackletCalculatorDisplaced.cc [hunk 1]
```diff
@@ -409,43 +426,119 @@ bool TrackletCalculatorDisplaced::LLLSeeding(const Stub* innerFPGAStub,
...
+                 phiderapprox,
+                 zderapprox,
+                 phiprojdiskapprox,
+                 rprojdiskapprox,
+                 phiderdiskapprox,
+                 rderdiskapprox);
+
+  if (isfinite(phi0approx))
+    phi0approx = angle0to2pi::make0To2pi(phi0approx);
+  assert(!isfinite(phi0approx) || fabs(phi0approx) < 2 * M_PI);
```
Comment (aehart) [inline L1Trigger/TrackFindingTracklet/src/TrackletCalculatorDisplaced.cc]: @gabhijith I would suggest moving all the calls to `angle0to2pi::make0To2pi` inside the `approxtracklet` and `approxproj`/`approxprojdisk` functions. I also think the `assert` is unnecessary.

Also, I know checking whether φ<sub>0</sub> is finite was necessary at certain points during debugging, but is it still necessary? If not, then we should remove it.

These comments apply to all similar instances below.

File: L1Trigger/TrackFindingTracklet/src/TrackletCalculatorDisplaced.cc [hunk 2]
```diff
@@ -1500,14 +1746,203 @@ void TrackletCalculatorDisplaced::exacttracklet(double r1,
...
+               t,
+               z0,
+               halfRinv_0,
+               d0_0,  // added _0 version for high term calculations
+               toR_.at(i),
+               phiproj[i],
+               phider[i],
+               zproj[i],
+               zder[i]);
+    // phiproj[i] -= 0.171;
```
Comment (aehart) [inline L1Trigger/TrackFindingTracklet/src/TrackletCalculatorDisplaced.cc]: @gabhijith I would suggest removing this and similar comments, assuming we no longer have to apply this magic offset.

**Comment by aehart** [comment]
@tomalin @gabhijith I applied all the code formatting fixes from the instructions on the TWiki. Some of them affect the track quality code that was recently added, and I figured they might as well be included in this PR. I also rebased with the latest changes to the L1TK-dev-11_2_0_pre6 branch.

Other suggestions I made inline.

**Review by aehart — APPROVED** [review]
Thanks @gabhijith. This looks good to me now.

**Pull request merged**